.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,180 @@
+name: Create Linear Issue on Pull Request
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+        types: [opened]
+
+permissions:
+    pull-requests: write
+
+jobs:
+    create-linear-issue-on-pull-request:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
```

Comment:
Do we need to checkout the actual code repository?

Reply:
I guess maybe not. I'll remove it and see